theme/cdnjs.j2

```diff
@@ -0,0 +1,11 @@
+{% from 'cdnjs_hashes.j2' import cdnjs_hashes %}
+
+{% macro url(library, version, asset) -%}
```

Comment:
You can probably inline this macro as you aren't using it (and I understand it's `pub` by default? It's probably just collected into a Python dict in the j2 implementation)

Reply:
Yes; I split it off into its own macro because this is a fairly-reuseable template, and there's cases where it would make sense for a theme to use `url()` directly  :3

Reply:
Ah, alright 😁👍
You can resolve this comment, in that case :)
I'm checking out the other PR in the meantime :3